index.html

```diff
@@ -7,6 +7,6 @@
     <title>Welcome to potionfy</title>
 </head>
 <body>
-    
+    <h1>Welcome to potionfy!!. Have a nice stay!!</h1>
```

Comment:
Could you remove the 'have a nice stay' string and make the 'Welcome to potionfy' string red?

Reply:
Sure can do